Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 10, in <module>
    from socrates.set import REDIS 
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 10, in <module>
    from socrates.set import REDIS 
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 10, in <module>
    from socrates.set import REDIS 
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 10, in <module>
    from socrates.set import REDIS 
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 10, in <module>
    from socrates.set import *
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 10, in <module>
    from socrates.set import *
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 10, in <module>
    from socrates.set import *
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 10, in <module>
    from socrates.set import *
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import *
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import *
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import *
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import *
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import *
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import *
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import *
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import *
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import log, REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import log, REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import log, REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import log, REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from socrates.set import REDIS
ImportError: No module named socrates.set
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
ValueError: Attempted relative import in non-package
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
  File "elfin/socrates/set.py", line 9, in <module>
    level=logging.INFO
  File "/usr/lib/python2.7/logging/__init__.py", line 1528, in basicConfig
    hdlr = FileHandler(filename, mode)
  File "/usr/lib/python2.7/logging/__init__.py", line 901, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python2.7/logging/__init__.py", line 924, in _open
    stream = open(self.baseFilename, self.mode)
IOError: [Errno 2] No such file or directory: '/home/nightwish/log/run.log'
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
  File "elfin/socrates/set.py", line 9, in <module>
    level=logging.INFO
  File "/usr/lib/python2.7/logging/__init__.py", line 1528, in basicConfig
    hdlr = FileHandler(filename, mode)
  File "/usr/lib/python2.7/logging/__init__.py", line 901, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python2.7/logging/__init__.py", line 924, in _open
    stream = open(self.baseFilename, self.mode)
IOError: [Errno 2] No such file or directory: '/home/nightwish/log/run.log'
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
  File "elfin/socrates/set.py", line 9, in <module>
    level=logging.INFO
  File "/usr/lib/python2.7/logging/__init__.py", line 1528, in basicConfig
    hdlr = FileHandler(filename, mode)
  File "/usr/lib/python2.7/logging/__init__.py", line 901, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python2.7/logging/__init__.py", line 924, in _open
    stream = open(self.baseFilename, self.mode)
IOError: [Errno 2] No such file or directory: '/home/nightwish/log/run.log'
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
  File "elfin/socrates/set.py", line 9, in <module>
    level=logging.INFO
  File "/usr/lib/python2.7/logging/__init__.py", line 1528, in basicConfig
    hdlr = FileHandler(filename, mode)
  File "/usr/lib/python2.7/logging/__init__.py", line 901, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python2.7/logging/__init__.py", line 924, in _open
    stream = open(self.baseFilename, self.mode)
IOError: [Errno 2] No such file or directory: '/home/nightwish/log/run.log'
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
  File "elfin/socrates/set.py", line 9, in <module>
    level=logging.INFO
  File "/usr/lib/python2.7/logging/__init__.py", line 1528, in basicConfig
    hdlr = FileHandler(filename, mode)
  File "/usr/lib/python2.7/logging/__init__.py", line 901, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python2.7/logging/__init__.py", line 924, in _open
    stream = open(self.baseFilename, self.mode)
IOError: [Errno 2] No such file or directory: '/home/nightwish/log/run.log'
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
  File "elfin/socrates/set.py", line 9, in <module>
    level=logging.INFO
  File "/usr/lib/python2.7/logging/__init__.py", line 1528, in basicConfig
    hdlr = FileHandler(filename, mode)
  File "/usr/lib/python2.7/logging/__init__.py", line 901, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python2.7/logging/__init__.py", line 924, in _open
    stream = open(self.baseFilename, self.mode)
IOError: [Errno 2] No such file or directory: '/home/nightwish/log/run.log'
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
  File "elfin/socrates/set.py", line 9, in <module>
    level=logging.INFO
  File "/usr/lib/python2.7/logging/__init__.py", line 1528, in basicConfig
    hdlr = FileHandler(filename, mode)
  File "/usr/lib/python2.7/logging/__init__.py", line 901, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python2.7/logging/__init__.py", line 924, in _open
    stream = open(self.baseFilename, self.mode)
IOError: [Errno 2] No such file or directory: '/home/nightwish/log/run.log'
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/nightwish/elfin/scripts/get_access_token.py", line 12, in <module>
    from ..socrates.set import REDIS
  File "elfin/socrates/set.py", line 9, in <module>
    level=logging.INFO
  File "/usr/lib/python2.7/logging/__init__.py", line 1528, in basicConfig
    hdlr = FileHandler(filename, mode)
  File "/usr/lib/python2.7/logging/__init__.py", line 901, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python2.7/logging/__init__.py", line 924, in _open
    stream = open(self.baseFilename, self.mode)
IOError: [Errno 2] No such file or directory: '/home/nightwish/log/run.log'
